src/pages/Home.tsx

```diff
@@ -27,8 +27,18 @@ export default function Home() {
       throw error;
     }
   }, []);
+  const [cancelWaitForChecks, setCancelWaitForChecks] = useState<() => void>(() => {
+    return () => {};
+  });
+  useEffect(() => {
+    return () => {
+      cancelWaitForChecks();
+    };
+  }, [cancelWaitForChecks]);
```

Comment:
Can we simplify it to this?

```suggestion
    return cancelWaitForChecks;
  }, [cancelWaitForChecks]);
```

Reply:
this one can be simplified 👍